### DISCUSSION TITLE
Pass multiple values of headers using list or tuple

### DISCUSSION THREAD

**Discussion by GeekDuanLian** [encode/httpx#1664] [Ideas]
For example, the `Accept` header may have multiple values.
Can pass multiple values like this:
`s.get('xxx', headers={'Accept': ['text/html', 'application/json']})`
Which is equivalent to:
`s.get('xxx', headers={'Accept': 'text/html, application/json'})`
But more pythonic.

**Comment by lovelydinosaur** [comment]
I'm probably in favour of this, yes.

There are a few cases which *shouldn't* concatenate, but should instead result in multiple headers. Eg... https://github.com/bnoordhuis/mozilla-central/blob/master/netwerk/protocol/http/nsHttpHeaderArray.h#L185

**Reply by GeekDuanLian** [reply]
Client requests don’t seem to be so similar, they should all be placed in one.
